Cargo.toml

```diff
@@ -213,6 +213,9 @@ security-framework = "0.4"
 atty = "0.2"
 nix = "0.17.0"
 
+[build]
+jobs = 2 
```

Comment:
This particular setting is not valid here I think? Try at `.cargo/config.toml` on the host. https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure

Reply:
This is weird because it works with this but not without? :) I'll try your suggestion too. Computers. Why do we even? :)

Reply:
Hmmmm I think it's better if we don't have it in the cargo.toml anywho since some folks have more than 2 threads to use. :)

Reply:
Yeah - I was planning to add some configurable "stuff" to do this but I am happy to leave it in the runner for now.